docs/pages/develop/user-interface/splash-screen-and-app-icon.mdx

```diff
@@ -4,153 +4,123 @@ description: Learn how to add a splash screen and app icon to your Expo project.
+- Use a 1024x1024 image.
+- Use a **.png** file.
+- Use a transparent background.
+- Use the [Figma template](https://www.figma.com/community/file/1466490409418563617) to create your splash screen icon.
 
 </Collapsible>
 
 </Step>
 
 <Step label="2">
 
-### Export the splash image as a .png
+### Export the splash icon as a .png
 
-After creating a splash screen, export it as a **.png** and save it in the **assets/images** directory. By default, Expo uses **splash.png** as the file name. If you decide to change the name of your splash screen file, make sure to use that in the next step.
+After creating a splash screen icon, export it as a **.png** and save it in the **assets/images** directory. By default, Expo uses **splash-icon.png** as the file name. If you decide to change the name of your splash screen file, make sure to use that in the next step.
 
-> **Note:** **Currently, only .png images are supported** to use as a splash screen in an Expo project. If you use another image format, making a production build of your app will fail.
+> **Note:** **Currently, only .png images are supported** to use as a splash screen icon in an Expo project. If you use another image format, making a production build of your app will fail.
 
 </Step>
 
 <Step label="3">
 
-### Add the splash screen in app config
+### Configure the splash screen icon
 
-Open the app config and add the local path as the value of [`splash.image`](/versions/latest/config/app/#image) property to point to your new splash image.
+Open the app config file, under plugins set the corresponding properties as follows:
 
 ```json app.json
 {
-  "splash": {
-    /* @info */
-    "image": "./assets/images/splash.png"
-    /* @end */
+  "expo": {
+    "plugins": [
+      [
+        "expo-splash-screen",
+        {
+          /* @info The background color of the splash screen */
+          "backgroundColor": "#232323",
+          /* @end */
+          /* @info The path to the splash screen icon */
+          "image": "./assets/splash-icon.png",
+          /* @end */
+          /* @info The path to the splash screen icon for dark mode (optional) */
+          "dark": {
+            "image": "./assets/splash-icon-dark.png",
+            /* @end */
+            /* @info The background color of the splash screen for dark mode */
+            "backgroundColor": "#000000"
+            /* @end */
+          },
+          /* @info The width of the splash screen icon */
+          "imageWidth": 200
+          /* @end */
+        }
+      ]
+    ]
   }
 }
 ```
 
-You can test your new splash image by starting the development server with `npx expo start` and using a development build or Expo Go to preview it.
+To test your new splash screen, create a [preview](/build/eas-json/#preview-builds) or [production](/build/eas-json/#production-builds) build of your app.
```

Comment:
```suggestion
To test your new splash screen, create a [preview](/build/eas-json/#preview-builds) or [production](/build/eas-json/#production-builds) build of your app.

Learn more about [configurable splash screen properties](/versions/latest/sdk/splash-screen/#configurable-properties).
```

- At this point of the user journey, development, preview, or production builds haven't been introduced. It can lead to confusion. Might be good to link to the Preview and Production builds EAS tutorial chapters here.
- Can you use the BoxLink component here to point to the config plugin properties for the Splash Screen API?